src/actions/check-solution/index.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as core from '@actions/core';
+import { ActionLogger, AuthHandler, AuthKind, getWorkingDirectory, PacRunner } from '../../lib';
+import path = require('path');
+import fs = require('fs-extra');
+
+core.startGroup('check-solution:');
+const workingDir = getWorkingDirectory('working-directory', false);
+const solutionPathCandidate = core.getInput('path', { required: true });
+const solutionPath = path.isAbsolute(solutionPathCandidate) ? solutionPathCandidate : path.resolve(workingDir, solutionPathCandidate);
+fs.ensureFileSync(solutionPath);
```

Comment:
should be .ensureDirSync()

Reply:
This is a file path, not a directory path.
That being said, this is still wrong as 'ensureFileSync()' will actually create the file if it's missing, which we do not want.
Removed this line, and added the error checking below.